### PR TITLE
655 datafiles

### LINKED ISSUE #655: Data files should be made available
Installing via pip doesn't include data files

These should be bundled with pip (with hints for where to find them), or available to download via a separate install.

### PR DESCRIPTION
**Pull request by tyronerees**
#### Description of Work

Fixes #655 

#### Testing Instructions

1.  Check that running `fitbenchmarking` works.
2.  Check the updates to the docs make sense.
3.  Check the downloads linked to on the docs are there

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?

### CODE REVIEW

File: fitbenchmarking/cli/main.py [hunk 1]
```diff
@@ -38,20 +38,27 @@ def get_parser():
...
+    $ fitbenchmarking fitbenchmarking/examples/benchmark_problems/NIST/*
     $ fitbenchmarking -o examples/options_template.ini \
 examples/benchmark_problems/simple_tests examples/benchmark_problems/Muon '''
 
     parser = argparse.ArgumentParser(
         prog='FitBenchmarking', add_help=True, epilog=epilog,
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
+    root = os.path.dirname(inspect.getfile(fitbenchmarking))
+    print("root = {}".format(root))
```
Comment (wathen) [inline fitbenchmarking/cli/main.py:50]: please remove print statement

File: fitbenchmarking/cli/main.py [hunk 2]
```diff
@@ -38,20 +38,27 @@ def get_parser():
 
     epilog = '''Usage Examples:
```
Comment (wathen) [inline fitbenchmarking/cli/main.py:39]: The usage examples need to include `-p` in both and I think there should be a bit of text before the examples stating that this is run with the source code downloaded.

**Review by wathen — CHANGES_REQUESTED** [review]
Looks good, just a few doc things. Also, I think we need to update the [running docs](https://fitbenchmarking.readthedocs.io/en/latest/users/running.html) to reflect the new changes

File: docs/source/users/running.rst [hunk 3]
```diff
@@ -4,24 +4,41 @@
...
+   fitbenchmarking
+
+will run the NIST test example on SciPy minmizers.
+
+Running alternative problems
+----------------------------
+
+Other problems written in a :ref:`supported file format <problem_def>`
+can be analyzed with FitBenchmarking by
+passing the path using the `--problem-sets` (or `-p`) option.
```
Comment (wathen) [inline docs/source/users/running.rst:20]:
```suggestion
passing the path using the ``--problem-sets`` (or ``-p``) option.
```

File: examples/benchmark_problems/NIST/META.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+NIST
```
Comment (wathen) [inline examples/benchmark_problems/NIST/META.txt:1]: Is this file needed?

**Review by wathen — CHANGES_REQUESTED** [review]
Just a few comments but looks great in general. Once the PR has gone in will we update the pypi version?

**Comment by tyronerees** [comment]
I think we should wait until #664 goes in too, then update pypi to 0.1.dev2, do some final checks to see all works as expected, and then release 0.1.0

**Review by wathen — APPROVED** [review]

**Pull request merged**